modules/rank-eval/build.gradle

```diff
@@ -23,3 +23,12 @@ esplugin {
   hasClientJar = true
 }
 
+integTestCluster {
+  // Modules who's integration is explicitly tested in integration tests
+  module project(':modules:lang-mustache')
+}
+
+run {
```

Comment:
The IT test where running without addind this run section, however I saw in the commit where you moved the reindex tests that you added them. I'm curious if/when this section is used. Is it for the case when I start clusters from the command line from one of the plugin/module directories or is it used by something else? Or can I safely remove it in my case?

Reply:
It is used for `./gradlew -p modules/rank-eval run`. I try to keep the `integTestCluster` and `run` configurations the same.